src/main/java/no/unit/nva/cristin/projects/ProjectPresentation.java

```diff
@@ -1,15 +1,19 @@
 package no.unit.nva.cristin.projects;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import java.util.ArrayList;
 import java.util.List;
 
+@JsonPropertyOrder({"cristin_project_id", "main_language"})
```

Comment:
You can make those constants since you are re-using them

Reply:
Should i create constants for all strings used in annotations across all classes using them?

Reply:
Maybe they should be if they are used in many classes

Reply:
I get this error when trying to put the names in variables: "Attribute value must be constant". Maybe you have to declare them directly as strings as is already done?

Reply:
wait, i figured it out, using static final it does not complain anymore. Will fix later before merging